beacon-chain/db/kv/blocks_test.go

```diff
@@ -857,7 +869,7 @@ func TestStore_BlocksBySlot_BlockRootsBySlot(t *testing.T) {
 			require.NoError(t, err)
 			assert.Equal(t, true, proto.Equal(wantedPb, retrieved0Pb), "Wanted: %v, received: %v", retrievedBlocks[0], wanted)
 			wanted = b3
-			if b3.Version() >= version.Bellatrix {
+			if b3.Version() >= version.Bellatrix && b3.Version() < version.EPBS {
```

Comment:
Don't we need to add then a test for epbs?

Reply:
I'm unclear what you meant. We have an EPBS test in `blockTests`. These changes fix the previous behavior where the test always assumes the saved block is blinded for b3.